### DISCUSSION TITLE
Site Tree broken in 2.5.282

### DISCUSSION THREAD

**Discussion by sn4k3s** [requarks/wiki#5295] [Error / Bug Report]
Hi there, 
since 2.5.282 nothing happens if you click on the sites in the site tree (static navigation works fine). There are no errors in docker logs.
Apparently, the function get called for this don't return anything anymore. 
Downgrade to version 2.5.281 fixed the problem for the moment. 

Environment:
Debian 10
docker-ce & docker-ce-cli 20.10.15
image: requarks/wiki:2

**Comment by poltrian** [comment]
Same here in a Linux installation without docker

**Comment by outlawdengler** [comment]
Same here!
Ubuntu and Docker.

**Comment by luismanson** [comment]
Yes, same here. devtools shows a Js error:
`
TypeError: Cannot read properties of undefined (reading 'locale')
`
I was using another language, cleaning browser cache and setting English again as default did not make a difference.

**Reply by JoogsWasTaken** [reply]
Can confirm. Rolling back to 2.5.281 works. Interestingly, Chromium (101.0.4951.64) and Firefox (100.0.1) seem to report slightly different errors on my machine. I attached the logs (although obfuscated). Maybe it'll help.

[log-wikijs-tree-chromium.log](https://github.com/requarks/wiki/files/8705868/log-wikijs-tree-chromium.log)
[log-wikijs-tree-firefox.log](https://github.com/requarks/wiki/files/8705869/log-wikijs-tree-firefox.log)

**Comment by jlucki** [comment]
Seeing the same issue. This commit seems to be the cause:

https://github.com/requarks/wiki/commit/5590507ef331924a564c2399f3b42a32cbe7a6d3

Reverting this change locally, the menu is working again.

**Reply by peteruithoven** [reply]
Seems like the most logical candidate indeed. I've added a comment to the pull request:  
https://github.com/requarks/wiki/pull/5286#issuecomment-1128649955

**Comment by myml** [comment]
Sorry, I am looking for the problem.

**Comment by myml** [comment]
I have submit a fix  https://github.com/requarks/wiki/pull/5304

**Reply by sn4k3s** [reply]
Unfortunately, it's still broken after the fix :/

**Reply by jlucki** [reply]
> Unfortunately, it's still broken after the fix :/

There hasn't been a new release. Unless you've compiled the project files with the change in place you'll have to wait for 2.5.283. I tried out the fix locally and it worked fine for me.

**Comment by george-viaud** [comment]
Same problem here immediately after upgrading.  When will 2.5.283 release with this fix?

**Comment by umarch06** [comment]
@JoogsWasTaken  How can I revert to 2.5.281? I installed wiki.js by using this tutorial:

https://docs.requarks.io/install/ubuntu

**Comment by iroddis** [comment]
@umarch06 You can specify the full version in the docker image specification. So any place you used `requarks/wiki:2`, you can use `requarks/wiki:2.5.281`. Once 2.5.283 has been released (hopefully with the fix), you can go back to `requarks/wiki:2` to keep tracking the current version.

**Comment by peteruithoven** [comment]
[v2.5.283](https://github.com/requarks/wiki/releases/tag/v2.5.283) was just released with the fix.

**Comment by luismanson** [comment]
I confirm it's working. Thanks!

**Comment by chiuczek** [comment]
I have a similar problem — items in the site tree do not work when clicked. I updated to 2.5.283 but the problem is still happening, so maybe it is a separate issue?

It only occurs for pages which match folders ("home" pages for their folder). Any folders which don't have a page with a matching name will work fine. When I look at the dev tools there is this error in the logs:
```
TypeError: e.item is undefined                               app.js:478:11699
    i https://wiki.intellisense.io/_assets/js/theme0.js?1652678279:2
    _render https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    i https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    get https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    run https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    mn https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    re https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    Xt https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    Kt https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    Ut https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    qt https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    _render https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    i https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    get https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    run https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    mn https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    re https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
    Xt https://wiki.intellisense.io/_assets/js/app.js?1652678279:478
```

Firefox 100.0.2 on Ubuntu, but colleagues on other browsers/OS see the same issue

**Reply by chiuczek** [reply]
Downgrading to 2.5.281 makes things work fine, so I think it _is_ the same issue as discussed here

**Reply by peteruithoven** [reply]
I can't reproduce the issue anymore in `2.5.283`. 

I created pages at `/folderpage` and `/folderpage/subpage` and it I can click on both. 

Are you 100% you where testing `2.5.283`? Did you clear your cache etc?
